src/Command/Playercount.js

```diff
@@ -7,57 +7,20 @@ const messages = {
     'help': 'You can use `!playercount` to show the players connected to the network or some of the games.\nYou can use any of the following names: {names}'
 };
 
-const keyAlias = {
-    'hcf': 'hcfactions',
-    'anni': 'annihilation',
-    'civ-war': 'civwar',
-    'civcraft': 'civwar',
-    'minetheftauto': 'mta',
-    'gc': 'ghostcraft',
-    'ghost': 'ghostcraft',
-    'mine theft auto': 'mta',
-    'death by void': 'dbv',
-};
-
-// Map of key to name - also acts as a whitelist
-const gameNames = {
-    'all': 'Shotbow',
-    'annihilation': 'Annihilation',
-    'minez': 'MineZ',
-    'hcfactions': 'HCFactions',
-    'dbv': 'Death by Void',
-    'smash': 'SMASH',
-    'slaughter': 'Slaughter',
-    'civwar': 'Civ-War',
-    'gg': 'GG',
-    'mta': 'Mine Theft Auto',
-    'ghostcraft': 'GhostCraft',
-    'lobby': 'Lobby',
-};
-
-const roomToGame = {
-    '228706232133746688': 'annihilation',
-    '228706390883958784': 'dbv',
-    '228706280951382016': 'gg',
-    '368735807345000449': 'ghostcraft',
-    '228706314233184256': 'mta',
-    '228706218351394816': 'minez',
-    '228706292913537027': 'slaughter',
-    '228706345807904768': 'smash',
-};
-
 const cacheTTL = 10 * 1000; // 10 seconds in milliseconds
 
 module.exports = Command.extend({
     commandName: 'playercount',
     https: null,
     cache: null,
     cacheKey: 'serverlist',
+    config: null,
     dependencies: {
         'discordClient': 'discordClient',
         'commandPrefix': 'commandPrefix',
         'https': 'https',
-        'Cache': 'cache'
+        'Cache': 'cache',
+        'config': 'config'
```

Comment:
This isn't necessary as dependencies are merged with their parent class.  Please remove it.